pkg/apis/v1alpha1/register.go

```diff
@@ -58,6 +58,8 @@ var (
 	ResourceAWSNeuron   v1.ResourceName = "aws.amazon.com/neuron"
 	ResourceHabanaGaudi v1.ResourceName = "habana.ai/gaudi"
 	ResourceAWSPodENI   v1.ResourceName = "vpc.amazonaws.com/pod-eni"
+	NVIDIAaccelerator                   = "nvidia"
```

Comment:
Is there an upstream naming convention in `v1`? If not, maybe we can create a type for Nvidia and AWS for AcceleratorManufacturer?

i.e.

```go
type AcceleratorManufacturer string
```